GameOfRunes/GameOfRunes/Types/PowerUpType.swift

```diff
@@ -99,6 +62,31 @@ enum PowerUpType: String, CaseIterable {
         }
     }
     
+    func activate(at position: CGPoint, with size: CGSize?, gameEngine: GameEngine) {
```

Comment:
but this is still doing composition right, not really inheritance like what waikay wants